src/omnicore/omnicore.h

```diff
@@ -95,7 +95,7 @@ enum TransactionType {
 
 #define MSC_PROPERTY_TYPE_INDIVISIBLE             1
 #define MSC_PROPERTY_TYPE_DIVISIBLE               2
-#define MSC_PROPERTY_TYPE_UNIQUE                  3
+#define MSC_PROPERTY_TYPE_UNIQUE                  5
```

Comment:
Is there any reason to do so?

Reply:
Honestly, I can't recall now!  I'd say it conflicted with another of my local experiments as the most likely reason...